readme.txt

```diff
@@ -4,7 +4,7 @@ Tags: license manager, license, license number, serial number, activation number
 Requires at least: 5.0
 Tested up to: 6.3
 Requires PHP: 5.6
-Stable tag: trunk

```

Comment:
@bappi Why was this changed?

Reply:
The reviewers don't allow trunk as the stable tag anymore, and they can just take down the plugin just because of that. So it's just a safety measure, and we need to update it with every new release